src/main/kotlin/com/cjbooms/fabrikt/generators/controller/ControllerGeneratorUtils.kt

```diff
@@ -78,6 +78,9 @@ object ControllerGeneratorUtils {
 
     fun controllerName(resourceName: String) = "$resourceName${ControllerType.SUFFIX}"
 
-    fun methodName(verb: String, isSingleResource: Boolean) =
+    fun methodName(op: Operation, verb: String, isSingleResource: Boolean) =
+        op.operationId ?: httpVerbMethodName(verb, isSingleResource)
```

Comment:
The `operationId` is treated as-is. Perhaps we should manipulate it by some sort of normalization, uncapitalize first letter, camel case,... Any thought?

Reply:
Agreed. Stick something in `NormalisedString` class or reuse `camelToSnake()`

Reply:
👍

Reply:
Going for `NormalisedString.camelCase()` as an attempt to be pragmatic here